great_expectations/validation_operators/validation_operators.py

```diff
@@ -19,6 +19,42 @@
 # NOTE: Abe 2019/08/24 : This is first implementation of all these classes. Consider them UNSTABLE for now.
 
 
+class ValidationOperatorResult(object):
+    # TODO this is a design sketch
+    def __init__(self):
+        raise NotImplementedError
+
+    def get_run_id(self):
```

Comment:
Since we'll soon allow assets to have different names inside the same overall run_name, I think this needs to adjust a bit. Maybe this would include the overall run_name and run_time components? In that case, I think it makes sense to have the validation results *also* available by asset_name (in addition to by expectation suite identifier)